### ISSUE TITLE
Weekly Status 2017-04-17

### ISSUE THREAD

**Issue by mrjoro** [ampproject/amphtml#8794]
Please add a comment to this issue with your weekly status update for the AMP Project, such as:

- your accomplishments over the last week
- what you're currently working on
- things you're blocked on
- anything else you think the entire community should know

We encourage updates from everyone who is actively participating in the AMP Project.

**Comment by mrjoro** [comment]
- merged #8783 to add OWNERS.yaml to most extensions directories
- created [new form](https://goo.gl/forms/T65peVtfQfEoDWeD3) for people to request membership in the AMP Project on GitHub (#8676) 
- #8667 to make the new issue template a bit more clear

**Comment by kmh287** [comment]
- Working on cleaning up `service.js` (more eyes would be appreciated) #8698 
- Working on PR for supporting binding to `src` attribute on `amp-list`

**Comment by lannka** [comment]
- Working on the "layout delay metric" for content elements (to be compared with ad elements). #7789
- Support "referrerPolicy" in amp-pixel. #8372

**Comment by dvoytenko** [comment]
* Rotation complete
* Debugging iOS GPU bugs
* CSS Animation support
* Ad analytics follow through for full analysis and launch

**Comment by jridgewell** [comment]
- Catching up with everything
- AMP Layers

**Comment by erwinmombay** [comment]
- out 2 days
- fixing owners tests
- migrate owners bot to new server

**Comment by edhollinghurst** [comment]
- AMP Timeago extension [#8557](https://github.com/ampproject/amphtml/pull/8557)

**Comment by rsimha-amp** [comment]
- Fixed issue #8188 (prevent experiment clean up errors) with PR #8448 
- OOO 4/6-4/14

**Comment by alanorozco** [comment]
- Audited ad network usage of `window.context` and reconciled missing APIs in new impl (#8790)
- Added call tracking example in ABE https://github.com/ampproject/amp-by-example/pull/736 and updated docs (#8563)
- Allowed `IframeMessagingClient` to support multiple callbacks for the same response type (#8591)
- Some CSS animation pausing research / hair-pulling.

**Comment by cramforce** [comment]
- `pump-early-frame` experiment should launch this week. Better time to first paint.
- decreased time until we install ServiceWorker from the cache #8679
- started working on IO talk

**Comment by muxin** [comment]
- Fix image-lighbox image does not resize on orientation change issue
- Resolve more flake tests in error logging
- Working on bug fix that some browsers cannot apply fromCharCode

**Comment by camelburrito** [comment]
- Release/On Call rotation 
- AMPStart
- - Preview modals on home page
- - Investigate ways to do CSS configurations
- - Set up wordpress for porting themes.

**Comment by aghassemi** [comment]
- `amp-sort` design, prototype and presentation #8691 
- Fix for disable3pfallback https://github.com/ampproject/amphtml/pull/8753 
- A11Y chat with Rob Dodson to review https://github.com/ampproject/amphtml/pull/6333 and push for a fix to https://bugs.chromium.org/p/chromium/issues/detail?id=667833
- PositionObserver discussions and 60FPS prototype for scroll-bound animations
- Debugging (mostly head-scratching) an iOS issue caused by `overflow-scrolling: touch`

**Comment by levidurfee** [comment]
* Looked into #8688 
* Browse the issues and see if there is anything I can help with

**Comment by zhouyx** [comment]
- Added support for `waitFor` value in `visibilitySpec`
- Worked on providing position API to ad (still in discussion)
